app/assets/stylesheets/cart_index.css.scss

```diff
@@ -5,3 +5,12 @@
   color: white;
   text-align: center;
 }
+
+.checkout {
+  margin-top: 20px;
+  float: right;
+}
+
+.total_price {
+  text-align: right;
+}
```

Comment:
styling for the Total: Price 

I feel like there is probably a better way to do this. I had to use a bunch of TD's to make it position correctly. It looks ok to the user though